gqlauth/migrations/0002_alter_userstatus_options.py

```diff
@@ -1,8 +1,17 @@
 # Generated by Django 4.1.7 on 2023-05-07 15:38
 
+from django.contrib.auth import get_user_model
 from django.db import migrations
 
 
+def set_userstatus(apps, schema_editor):
```

Comment:
A way to test this would be to disconnect [this](https://github.com/nrbnlulu/strawberry-django-auth/blob/42b996af8aacfc6e152afeb2bf87e46a99c026ee/gqlauth/user/signals.py#L6) receiver, create a user and see if he has a status.